src/jquery.mask.js

```diff
@@ -467,6 +472,10 @@
         return this.data('mask').getCleanVal();
     };
 
+    $.fn.maskedVal = function(val) {
```

Comment:
But... for the angulas propose... I think it would be more optimal your suggestion. But... I would love to see the suggested change above on this PR, what do you think?